### PR TITLE
INTR-194 Update one up card styling

### PR DESCRIPTION
**Pull request by 0RWilliams**
Updated 'One up card' to meet the new designs

- No date shown
- Only show comments if there are comments to show
- Optional ribbon (shown only when `ribben_text` is provided.

Full size:
image

Mobile:
image

- [x] JIRA ticket referenced in title
- [x] Title is clear and concise
- [x] Description gives any relevant detail
- [x] Tests are up to date
- [x] Documentation is up to date

### CODE REVIEW

File: src/dw_design_system/dwds/components/one_up_card.html [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
+                             src="{% static '/images/dwds/icons/comment_icon.svg' %}"
+                             alt="Comment bubble icon.">
+                        <span>{{ comment_count }} Comment{{ comment_count|pluralize }}</span>
+                    </p>
                 </div>
             {% endblock card_footer %}
         </div>
+        <div class="dwds-card--image">
+            {% if thumbnail %}
+                {% image thumbnail fill-320x200 %}
```
Comment (alsyx) [inline src/dw_design_system/dwds/components/one_up_card.html:42]: Is there a case we might not have a thumbnail and need a fallback image?

File: src/dw_design_system/dwds/components/one_up_card.html [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
-                <div class="card-footer">
-                    {% block card_footer_body %}
-                        <time datetime="{{ date|date:'Y-m-d H:i' }}">{{ date|date:'j F Y' }}</time>
-                    {% endblock card_footer_body %}
+                <div class="dwds-card--footer">
+                    <p>
+                        <img class="comment_icon"
+                             src="{% static '/images/dwds/icons/comment_icon.svg' %}"
+                             alt="Comment bubble icon.">
+                        <span>{{ comment_count }} Comment{{ comment_count|pluralize }}</span>
```
Comment (alsyx) [inline src/dw_design_system/dwds/components/one_up_card.html:35]: I think if there aren't any comments we shouldn't display comment count.

Comment (0RWilliams) [reply src/dw_design_system/dwds/components/one_up_card.html:35]: Yeah, had that in at one point, must have removed it, and forgot to put it back in, done.

File: src/dw_design_system/dwds/components/one_up_card.html (hunk 1 above)
Comment (0RWilliams) [reply src/dw_design_system/dwds/components/one_up_card.html:42]: It isn't specified on the task, but considering the other cards, probably no harm adding it.

**Review by alsyx — APPROVED** [review]

File: src/dw_design_system/dwds/components/one_up_card.scss [hunk 3]
```diff
@@ -1,13 +1,28 @@
+@media only screen and (max-width: 828px) {
```
Comment (SamDudley) [inline src/dw_design_system/dwds/components/one_up_card.scss:1]: I don't like the use of a magic pixel number here and I still feel that container queries are the better approach here. I've not used container queries a bunch so I'll have a tinker and come up with an example.

File: src/dw_design_system/dwds/components/one_up_card.scss [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+    .card-content{
+        order: 1;
+    }
+    .dwds-card--image{
+        order: 0;
+    }
+}
+
+@media (min-width: 828px) {
```
Comment (SamDudley) [inline src/dw_design_system/dwds/components/one_up_card.scss:10]: Same as above.

File: src/dw_design_system/dwds/components/one_up_card.scss [hunk 5]
```diff
@@ -20,13 +35,20 @@
...
-    p{
-        color: var(--dbt-light-blue)
+    .dwds-card--footer{
+        p{
+            padding: var(--space);
+            .comment_icon{
+                vertical-align: text-bottom;
+            }
+        }
     }
```
Comment (SamDudley) [inline src/dw_design_system/dwds/components/one_up_card.scss:45]: I would have expected the padding to be handled in the base card?

File: src/dw_design_system/dwds/components/one_up_card.scss [hunk 6]
```diff
@@ -20,13 +35,20 @@
...
-        width: 100%;
-        border: var(--border-width) solid var(--color-border);
+   
+    .dwds-card--image{
+        img {
+            width: 100%;
+            height: 100%;
+            vertical-align: text-bottom;
+        }
     }
```
Comment (SamDudley) [inline src/dw_design_system/dwds/components/one_up_card.scss:53]: Again, I would have thought this to be a base card concern.

Comment (0RWilliams) [reply src/dw_design_system/dwds/components/one_up_card.scss:53]: Removed, brought  up a layer, removing it's parent div.

**Review by SamDudley — APPROVED** [review]

**Pull request merged**